### PR TITLE
feat: move from Travis to Github Actions

### PR DESCRIPTION
**Pull request by svanherk**
Also deleting my test action `main.yml` and `test-aws.js`.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,19 +1,9 @@
 # navigation
-[![Published on webcomponents.org](https://img.shields.io/badge/webcomponents.org-published-blue.svg)](https://www.webcomponents.org/element/BrightspaceUI/navigation)
```
Comment (svanherk) [inline README.md:2]: This README needs a lot of cleanup (it still has all the webcomponents.org comments) but I'll look at that separately, wanted to get this up quickly so we can compare visual diff in its own workflow versus in CI.

File: .travis.yml [hunk 2]
```diff
@@ -1,50 +0,0 @@
...
-      - OWNER_NAME=BrightspaceUI
-      - REPO_NAME=navigation
-    deploy:
-      provider: releases
-      api_key: "$GITHUB_RELEASE_TOKEN"
-      on:
-        tags: true
-env:
-  global:
-  # GITHUB_RELEASE_TOKEN
```
Comment (dlockhart) [inline .travis.yml:49]: I'll remove this token from the bot account.

File: .github/workflows/release.yml [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+          persist-credentials: false
+      - name: Setup Node
+        uses: Brightspace/third-party-actions@actions/setup-node
+      - name: Install dependencies
+        run: npm install
+      - name: Unit Tests (Headless)
+        run: npm run test:polymer:local
+      - name: Semantic Release
+        uses: BrightspaceUI/actions/semantic-release@master
+        with:
```
Comment (dlockhart) [inline .github/workflows/release.yml:25]: This wasn't published to NPM before, so you'll want `NPM: false` here to maintain that.

Comment (svanherk) [reply .github/workflows/release.yml:25]: Ah, I figured that was the default

Comment (dlockhart) [reply .github/workflows/release.yml:25]: You're right it is, ignore me.

**Review by dlockhart — APPROVED** [review]

File: package.json [hunk 4]
```diff
@@ -12,15 +12,11 @@
...
-    "test": "npm run lint && npm run test:polymer:local",
-    "test:diff": "mocha ./test/**/*.visual-diff.js -t 40000",
-    "test:diff:golden": "mocha ./test/**/*.visual-diff.js -t 40000 --golden",
-    "test:diff:golden:commit": "commit-goldens"
+    "test": "npm run lint && npm run test:polymer:local"
   },
   "author": "D2L Corporation",
   "license": "Apache-2.0",
   "devDependencies": {
-    "@brightspace-ui/visual-diff": "^1",
```
Comment (jeroach) [inline package.json:23]: Are there no visual diff tests in this repo? I didn't expect to see this removed as a dev dependency

Comment (dlockhart) [reply package.json:23]: The GitHub Action for visual diff installs all the dependencies it needs -- which saves devs and CI having to install them over and over again for no reason. 🎉

Comment (jeroach) [reply package.json:23]: Nice!

**Review by jeroach — APPROVED** [review]

**Pull request merged**